onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -53,12 +53,25 @@ def process_tableau_data(data, xform):
     with the column header fields for the same form.
     Handles Flattenning repeat data for tableau
     """
+
+    def get_xpath(key, nested_key):
+        val = nested_key.split('/')
+        nested_key_diff = val[len(key.split('/')):]
+        xpaths = key + f'[{index}]/' + '/'.join(nested_key_diff)
+        return xpaths
+
+    def unpack_data(key, value, data=None, xpaths=None):
+        if isinstance(value, str) and data:
+            choices = value.split(" ")
+            for choice in choices:
+                xpaths = f'{key}/{choice}'
+                data[xpaths] = choice
```

Comment:
Do we want to rename this function to something like `update_data` since we are updating the data dictionary in the function?

Reply:
Renamed that to be more descriptive. Also included a docstring for that function